code/modules/uplink/uplink_items/device_tools.dm

```diff
@@ -232,9 +232,11 @@
 
 /datum/uplink_item/device_tools/hypnotic_grenade
 	name = "Hypnotic Grenade"
-	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers."
+	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers. \
+		Be warned however, that this grenade is finnicky to use and utilize. \
+		Ensure proper protection and instant uttering of hypnotic command before triggering the grenade."
```

Comment:
```suggestion
	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers. \
		Equipment to protect yourself from the effects are not provided. Ensure you are ready before pulling the pin."
```
I don't think the wording as you provided it works for the in-character uplink, in fact I think your wording is actively misleading (it sounds like you should say the command before triggering the grenade).